[2024-05-28T03:09:25.897+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_install_dbt_utils scheduled__2024-05-26T08:00:00+00:00 [queued]>
[2024-05-28T03:09:25.906+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: franko_lookup_ingest.ti_install_dbt_utils scheduled__2024-05-26T08:00:00+00:00 [queued]>
[2024-05-28T03:09:25.906+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T03:09:25.907+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-05-28T03:09:25.907+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-05-28T03:09:25.917+0000] {taskinstance.py:1304} INFO - Executing <Task(DockerOperator): ti_install_dbt_utils> on 2024-05-26 08:00:00+00:00
[2024-05-28T03:09:25.921+0000] {standard_task_runner.py:55} INFO - Started process 745 to run task
[2024-05-28T03:09:25.923+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'franko_lookup_ingest', 'ti_install_dbt_utils', 'scheduled__2024-05-26T08:00:00+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/challenge_dag.py', '--cfg-path', '/tmp/tmp9uz7vosj']
[2024-05-28T03:09:25.923+0000] {standard_task_runner.py:83} INFO - Job 276: Subtask ti_install_dbt_utils
[2024-05-28T03:09:25.968+0000] {task_command.py:389} INFO - Running <TaskInstance: franko_lookup_ingest.ti_install_dbt_utils scheduled__2024-05-26T08:00:00+00:00 [running]> on host a64b6410e10e
[2024-05-28T03:09:26.028+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=franko_lookup_ingest
AIRFLOW_CTX_TASK_ID=ti_install_dbt_utils
AIRFLOW_CTX_EXECUTION_DATE=2024-05-26T08:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-26T08:00:00+00:00
[2024-05-28T03:09:26.055+0000] {docker.py:284} INFO - Starting docker container from image my_dbt
[2024-05-28T03:09:29.935+0000] {docker.py:354} INFO - [0m03:09:29  Running with dbt=1.5.11
[2024-05-28T03:09:30.787+0000] {docker.py:354} INFO - [0m03:09:30  Installing dbt-labs/dbt_utils
[2024-05-28T03:09:31.893+0000] {docker.py:354} INFO - [0m03:09:31  Installed from version 1.0.0
[2024-05-28T03:09:31.894+0000] {docker.py:354} INFO - [0m03:09:31  Updated version available: 1.1.1
[2024-05-28T03:09:31.896+0000] {docker.py:354} INFO - [0m03:09:31
[2024-05-28T03:09:31.896+0000] {docker.py:354} INFO - [0m03:09:31  Updates available for packages: ['dbt-labs/dbt_utils']                 
Update your versions in packages.yml, then run dbt deps
[2024-05-28T03:09:33.167+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=franko_lookup_ingest, task_id=ti_install_dbt_utils, execution_date=20240526T080000, start_date=20240528T030925, end_date=20240528T030933
[2024-05-28T03:09:33.217+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-05-28T03:09:33.292+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
